Sources/Fluent/Schema/Schema+Creator.swift

```diff
@@ -125,21 +125,33 @@ extension Schema {
         }
 
         // MARK: Relations
-
         public func parent<E: Entity>(
             _ entity: E.Type = E.self,
             optional: Bool = false,
             unique: Bool = false,
             default: NodeRepresentable? = nil
+            ) {
+            parent(customId: "\(entity.name)_id", 
+                   optional: optional, 
+                   unique: unique, 
+                   default: `default`)
+        }
+
+        public func parent(
+            customId: String,
```

Comment:
`idKey` might be a better name for this. `customId` seems like its referring to the id values.